README.md

```diff
@@ -56,14 +56,14 @@ Enter the Tuna directory
 ```
 cd MITuna
 ```
-Create a virtual envornment, and activate it (by sourcing its `activate` script)
+Activate the virtualenv and source the virtual env for subsequent use
 ```
-virtualenv -p python3.9 myvenv
```

Comment:
why are we going back?

Reply:
I'll just checkout this file from develop